web/src/components/molecules/Asset/AssetListTable/index.tsx

```diff
@@ -140,10 +133,27 @@ const AssetListTable: React.FC<AssetListTableProps> = ({
     },
   };
 
+  const AlertOptions = (props: any) => {
+    return (
+      <Space size={16}>
+        <a
+          onClick={props.onCleanSelected}
+          style={{ display: "flex", alignItems: "center", gap: "8px" }}>
```

Comment:
a lot of inline styles in nested elements, wouldn't be cleaner if used styled-components for these styles?

Reply:
Done